demisto_sdk/commands/common/tools.py

```diff
@@ -1279,7 +1279,11 @@ def get_dict_from_file(path: str,
             if path.endswith('.yml'):
                 return get_yaml(path, cache_clear=clear_cache), 'yml'
             elif path.endswith('.json'):
-                return get_json(path, cache_clear=clear_cache), 'json'
+                res = get_json(path, cache_clear=clear_cache)
```

Comment:
please update the docstrings, mentioning the method returns the dict or first item if the file contents are a list with a single dictionary